pyproject.toml

```diff
@@ -17,7 +17,7 @@ exclude = []
 [tool.poetry.dependencies]
 python = "^3.6"
 starlette = ">=0.12.0"
-prometheus_client = "^0.5.0"
+prometheus_client = ">=0.5.0 <1.0.0"
```

Comment:
There is a similar way of achieving the same using `^` operator: `^0.5`

Could you modify it to this?

Reply:
Hi I currently have it running with 0.7.1 so that should be fine. I'll make the requested change.

Reply:
Hi @perdy 
The problem still persists! I've been on vacation on the last two weeks so I'm only looking at it now. I have also found the relevant piece of [documnetation](https://poetry.eustace.io/docs/versions/) on poetry: Apparently version beginning with 0 are an exception and `^0.5` actually resolves to `>=0.5 <0.6` not `>=0.5.0 <1.0.0`. So this means that 0.7.1 is still excluded. Can the requirement be changed to `>=0.5.0 <1.0.0` or `>=0.5.0 <0.8.0`?